content/blog/high-utilization-in-systems/index.md

```diff
@@ -0,0 +1,92 @@
+
+**Context switching**
+
+It's no surprise that people are more productive when they can work on a task from start to finish without interruption, but as the utilization goes up, so does the amount of concurrent tasks people are tackling. Unfortunately switching context is _expensive_, energy and time-wise. This will not only decrease the throughput of the team, but also decrease the overall quality of the final product.
+
+**Communication**
+
+One of the first things that breaks down is communication. Asynchronous messages start taking longer to be answered or not answered at all in public and private channels. Emails or issues will tend to be forgotten until someone volunteers or gets "voluntold". You will notice less engagement in meetings, with people multitasking or not participating at all. This in turn decreases the amount of collaboration and trust among team members.
+
+Communication also includes critical areas such as:
+
+* Mentorship
+* Knowledge sharing
+* Code reviews
+* Documentation
+
+Among those, code reviews and mentorship will directly impact how long the cycle time takes in the short term. This might happen due to team members being blocked from delivering their pull requests or due to degraded code quality (shallow reviews) causing unnecessary bugs in production.
+
+Notice how all of those areas benefit the organization for the long-term when done right. It's not a coincidence that they are deprioritized.
+
+## Delivery
+
+If doing things this way impacts delivery speed and quality in the long run, why do we still see high utilization as a positive signal? My theory is that most of our industry is caught up playing short term games. 
+
+Leadership is expected to provide results in short-term cycles, hence management gets recognized for delivering projects in tight timelines and developers are rewarded when they are able to ship tasks as fast as possible. None of these behaviours are responsible for creating better products, teams or culture in the long run. This reminds me of a quote from Dave Packard:
+
+> “More Organisations Die Of Indigestion Than Starvation”
+
+This quote really resonates with me. We need teams focused on a small number of objectives that are aligned with the company vision. High utilization doesn't allow for strategic planning, getting teams caught up in a tactical mindset.
+
+## Things to look out for
+
+If you are not sure whether your team is under high utilization or not (you probably know) you can look for the following characteristics:
+
+* Team members working alone or in small teams due too many initiatives
+* Poor engagement in meetings
+* Asynchronous channels (Slack, email, etc) drying up in participation or being slow/ignored
+* Pull request lead cycle taking a long time
+    - PRs are taking longer to be opened (high complexity/lower quality in the codebase)
+    - PRs are taking longer to be reviewed (context switching) 
+* Increased amount of bugs in production
+* Knowledge silos
```

Comment:
Nit: I can _guess_ what a "knowledge silo" is, but IMO, it's not a good bullet point term as it requires a little bit of clarification.

Reply:
Leave it to me. :)